<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>nfyHDR</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="ldrGroup">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 15px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>HDRI Recovery</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Image Files</string>
     </property>
    </widget>
    <widget class="QListWidget" name="imageList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>231</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Input LDR images to be used while HDR recovery. The items should be ordered in exposure time ascending order.</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="smoothness">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Smoothness value to be used while recovering the response curve of the camera that the input LDR images are photographed.</string>
     </property>
     <property name="maximum">
      <double>99999.990000000005239</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="smoothnessLabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Smoothness value to be used while recovering the response curve of the camera that the input LDR images are photographed.</string>
     </property>
     <property name="text">
      <string>  Smoothness:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="resizeFactor">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>110</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Resize factor that will be used while downsampling the input LDR images to speed up the HDR recovery process. An initial value will be suggested when the images are loaded.</string>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="resizeFactorLabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Resize factor that will be used while downsampling the input LDR images to speed up the HDR recovery process. An initial value will be suggested when the images are loaded.</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string>Resize Factor:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveHDR">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Save HDR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createHDR">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Recover HDR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="listX">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Remove the selected image from image list</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="listUp">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Move the selected image up</string>
     </property>
     <property name="text">
      <string>ÊŒ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="listDown">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>170</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="toolTip">
      <string>Move the selected image down</string>
     </property>
     <property name="text">
      <string>v</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>20</width>
       <height>261</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <zorder>pushButton</zorder>
    <zorder>imageList</zorder>
    <zorder>smoothness</zorder>
    <zorder>smoothnessLabel</zorder>
    <zorder>resizeFactor</zorder>
    <zorder>resizeFactorLabel</zorder>
    <zorder>saveHDR</zorder>
    <zorder>createHDR</zorder>
    <zorder>listUp</zorder>
    <zorder>listDown</zorder>
    <zorder>listX</zorder>
    <zorder>line_2</zorder>
   </widget>
   <widget class="QGroupBox" name="tmoGroup">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 15px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>Tonemapping</string>
    </property>
    <widget class="QPushButton" name="tonemap">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Global
Tonemapping</string>
     </property>
    </widget>
    <widget class="QPushButton" name="viewTonemappedImage">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>View Tonemapped Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveTonemappedImage">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Tonemapped Image</string>
     </property>
    </widget>
    <widget class="QLabel" name="keyValueLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Key value that is going to be used while tonemapping the input HDR image.</string>
     </property>
     <property name="text">
      <string>Key Value:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="keyValueGlobal">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Key value that is going to be used while tonemapping the input HDR image.</string>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999.990000000000009</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.180000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="lMax">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The luminosity value that controls where the burning starts in output tonemapped LDR image.</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>340282300000000000985401294049535934735581184.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>340282300000000000985401294049535934735581184.000000000000000</double>
     </property>
    </widget>
    <widget class="QCheckBox" name="lMaxCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The luminosity value that controls where the burning starts in output tonemapped LDR image.</string>
     </property>
     <property name="text">
      <string>Lmax</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openHdrFile">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Open HDR Image File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="localTonemap">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Local
Tonemapping</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="sharpening">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Sharpening value that controls the sharpness of edges while applying local tonemapping method.</string>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999.990000000000009</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>8.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="sharpeningLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Sharpening value that controls the sharpness of edges while applying local tonemapping method.</string>
     </property>
     <property name="text">
      <string>Sharpening:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="threshold">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>130</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Threshold to be used while determining the local average luminosity of each pixel while applying local tonemapping method.</string>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999.990000000000009</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>0.005000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="thresholdLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Threshold to be used while determining the local average luminosity of each pixel while applying local tonemapping method.</string>
     </property>
     <property name="text">
      <string>Threshold:</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>20</width>
       <height>121</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="keyValueLocal">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Key value that is going to be used while tonemapping the input HDR image.</string>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999.990000000000009</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.180000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="keyValueLabel_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Key value that is going to be used while tonemapping the input HDR image.</string>
     </property>
     <property name="text">
      <string>Key Value:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="Quit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
